lib/std/tempfiles.nim

```diff
@@ -70,7 +81,16 @@ template randomPathName(length: Natural): string =
     res[i] = letters[state.rand(61)]
   res
 
-proc mkstemp*(prefix, suffix: string, dir = ""): tuple[fd: File, name: string] =
+proc createTempFile*(prefix, suffix: string, dir = ""): tuple[fd: File, name: string] =
```

Comment:
i wish we had more consistent conventions in std/os, but `name` is bad.
option 1: s/name/path/
option 2: s/name/filename/

i prefer option 1 here

Reply:
I agree